Packs/Troubleshoot/Scripts/IsDockerImageExists/IsDockerImageExists.yml

```diff
@@ -0,0 +1,31 @@
+args:
```

Comment:
this script should have `dependson` demisto rest api

Reply:
done